assets/js/modules/pagespeed-insights/index.js

```diff
@@ -20,14 +20,18 @@
  * WordPress dependencies
  */
 import { addFilter } from '@wordpress/hooks';
+import domReady from '@wordpress/dom-ready';
 
 /**
  * Internal dependencies
  */
 import './datastore';
-import { getModulesData, fillFilterWithComponent, createAddToFilter } from '../../util';
+import { fillFilterWithComponent } from '../../util';
+import Widgets from '../../googlesitekit/widgets';
```

Comment:
This should _always_ be imported from `googlesitekit-wdgets` instead (similar to how we import `Data` from `googlesitekit-data`).

Reply:
Updated.